dio/test/options_test.dart

```diff
@@ -270,4 +273,21 @@ void main() {
     assert(r3.uri.toString() == 'https://www.example.com/test');
     assert(r3.headers[Headers.contentTypeHeader] == null);
   });
+
+  test('#test forceConvert responseType', () async {
+    final dio = Dio(BaseOptions(
+      baseUrl: MockAdapter.mockBase,
+    )) //
+      ..httpClientAdapter = MockAdapter();
+    final expectedResponseData = <String, dynamic>{"code": 0, "result": "ok"};
+
+    final response = await dio.get<Map<String, dynamic>>('/test-force-convert');
+    expect(response.data, expectedResponseData);
```

Comment:
For this call, the generic T is `Map<String, dynamic>` and the responshould be forced to parse as json.